config/samples/dataplane_v1beta1_openstackdataplanenodeset_with_ipam.yaml

```diff
@@ -87,32 +50,11 @@ spec:
          # Default nic config template for a EDPM compute node
          # These vars are edpm_network_config role vars
          edpm_network_config_hide_sensitive_logs: false
+         edpm_network_config_template: templates/single_nic_vlans/single_nic_vlans.j2
```

Comment:
this is the default. drop it?

Reply:
Now that we have merged the new CRD design, I'll keep pushing forward with my other PR to change how this interface works anyway. 
https://github.com/openstack-k8s-operators/dataplane-operator/pull/394